### PR TITLE
Feature/latest boiler plate

### PR DESCRIPTION
**Pull request by IamMHC**

### CODE REVIEW

**Comment by IamMHC** [comment]
@artalat sir kindly check and merge

File: bluebase/storybook-native/storybook/storyLoader.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Auto-generated file created by react-native-storybook-loader
```
Comment (artalat) [inline bluebase/storybook-native/storybook/storyLoader.js]: we dont need to commit this file as it is auto generated. Add it to gitignore

File: bluebase/storybook/bluebase.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { BootOptions } from '@bluebase/core';
+import commonBootOptions from '../common/bluebase';
+import deepmerge from 'deepmerge';
+
+// TODO: Only for evaluation, remove this
+import DummyPlugin from './sample';
```
Comment (artalat) [inline bluebase/storybook/bluebase.ts]: remove dummy data

File: bluebase/storybook/bluebase.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+/**
+ * Add your platform specific configs here. 
+ * We keep all the universal (cross platform) configs in 
+ * the common folder, and extend them here.
+ */
+const bootOptions: Partial<BootOptions> = {
+
+	plugins: [
+		// TODO: Only for evaluation, remove this
+		DummyPlugin
```
Comment (artalat) [inline bluebase/storybook/bluebase.ts]: remove dummy plugin

File: bluebase/storybook/bluebase.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+	// 	wallpaper: {
+	// 		backgroundColor: 'white',
+	// 		resizeMode: 'cover',
+	// 		source: require('./../../assets/storybook/wallpaper.jpg'),
+	// 	},
+	// }
+};
+
+export default deepmerge(commonBootOptions, bootOptions);
```
Comment (artalat) [inline bluebase/storybook/bluebase.ts]: we dont use deepmerge anymore. find latest syntax

File: package.json [hunk 5]
```diff
@@ -43,21 +33,61 @@
...
+  "size-limit": [
+    {
+      "limit": "3 KB",
+      "webpack": false,
+      "path": "dist/**/*.js"
+    }
+  ],
   "engines": {
     "node": ">=8.9"
   },
```
Comment (artalat) [inline package.json]: move this to the top

File: package.json [hunk 6]
```diff
@@ -43,21 +33,61 @@
...
   "dependencies": {
+    "@babel/runtime": "^7.7.4",
```
Comment (artalat) [inline package.json]: remove this dep

File: package.json [hunk 7]
```diff
@@ -43,21 +33,61 @@
...
+    "@bluebase/core": "^5.3.1",
```
Comment (artalat) [inline package.json]: move to dev dep

File: package.json [hunk 8]
```diff
@@ -43,21 +33,61 @@
...
+    "deepmerge": "^4.2.2",
+    "expo": "^35.0.0",
+    "metro-react-native-babel-preset": "^0.57.0",
```
Comment (artalat) [inline package.json]: move to dev dep

File: src/index.ts [hunk 9]
```diff
@@ -2,7 +2,7 @@ import { createPlugin } from '@bluebase/core';
 
 export default createPlugin({
 	description: '<%= PROJECT_DESCRIPTION %>',
-	key: '<%= PROJECT_NAME %>',
-	name: '<%= PROJECT_TITLE %>',
```
Comment (artalat) [inline src/index.ts]: revert this??

File: package.json [hunk 10]
```diff
@@ -1,26 +1,16 @@
 {
-  "name": "<%= PROJECT_NAME %>",
-  "description": "<%= PROJECT_DESCRIPTION %>",
```
Comment (artalat) [inline package.json]: revert this?

**Pull request closed without merging**